CHANGES.rst

```diff
@@ -68,6 +68,12 @@ refpix
   to avoid failures for moderate-brightness sources due to extremely low
   throughput at the long wavelength end of MRS band 4C. [#8199]
 
+resample
+--------
+
+- Changed to use APE 14 interface for reproject, which also fixed a bug, and removed
```

Comment:
Forgive my ignorance, but what exactly is "APE 14"?

Reply:
I should have made it clearer for the APE 14 high-level API used here (i.e., pixel_to_world_values and world_to_pixel_values).
https://github.com/astropy/astropy-APEs/blob/main/APE14.rst